.github/workflows/fly-deploy.yml

```
@@ -1,18 +1,31 @@
       - uses: superfly/flyctl-actions/setup-flyctl@master
-      - run: flyctl deploy --remote-only
+      - run: flyctl deploy --remote-only --build-only --push --no-cache --image-label "deployment-${{ github.ref_name }}"
         env:
           FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+
+  prompt-service:
+    name: Push to Service (Test Environment)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send POST request
+        run: |
```

actionlint (1.7.4)

got unexpected character '#' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)